src/app/util/attribute-storage.h

```diff
@@ -270,7 +270,8 @@ CHIP_ERROR emberAfSetDeviceTypeList(chip::EndpointId endpoint, chip::Span<const
 //
 EmberAfStatus emberAfSetDynamicEndpoint(uint16_t index, chip::EndpointId id, const EmberAfEndpointType * ep,
```

Comment:
Please update the comments to this function indicating the use of the new argument.

Reply:
Sure.